modules/carousel/jetpack-carousel.php

```diff
@@ -502,9 +502,12 @@ function get_attachment_comments() {
 		 * @since 1.6.0
 		 */
 		do_action('jp_carousel_check_blog_user_privileges');
+		
+		$request_id = filter_input( INPUT_REQUEST, 'id' );
+		$request_offset = filter_input( INPUT_REQUEST, 'offset' );
 
-		$attachment_id = ( isset( $_REQUEST['id'] ) ) ? (int) $_REQUEST['id'] : 0;
-		$offset        = ( isset( $_REQUEST['offset'] ) ) ? (int) $_REQUEST['offset'] : 0;
+		$attachment_id = ( isset( $request_id ) ) ? (int) $request_id : 0;
+		$offset        = ( isset( $request_offset ) ) ? (int) $request_offset : 0;
```

Comment:
The real benefit of `filter_input` here would be to remove the checks at all because they are already performed, and instead use something like:
```
$attachment_id = filter_input( INPUT_REQUEST, 'id', FILTER_VALIDATE_INT );
$offset = filter_input( INPUT_REQUEST, 'offset', FILTER_VALIDATE_INT );
```
Otherwise IMO it's just meaningless change.